### PR TITLE
Add new breaking changes policy

### PR DESCRIPTION
**Pull request by maxh**
Update readme to explain new breaking changes policy.  Not sure about the specifics -- Shane should review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,3 +229,7 @@ then the polyfill will provide a console warning in browsers where these feature
 
 For running tests or building minified files, consult the
 [tooling information](http://www.polymer-project.org/resources/tooling-strategy.html).
+
+## Breaking changes
+
+When we make a potentially breaking change to polyfill's API surface (like a rename) we keep the old version around on another branch, deprecated, for 3 months, and ensure that there are console warnings that a change is pending. We also announce anything that isn't a bug fix on web-animations-changes@googlegroups.com (https://groups.google.com/forum/#!forum/web-animations-changes).
```
Comment (shans) [inline README.md]: lose 'on another branch'. We'll keep both versions in head. Otherwise this is good.

**Comment by maxh** [comment]
Done.  I tweaked the wording to make this more clear too.

File: README.md [hunk 2]
```diff
@@ -229,3 +229,7 @@ then the polyfill will provide a console warning in browsers where these feature
 
 For running tests or building minified files, consult the
 [tooling information](http://www.polymer-project.org/resources/tooling-strategy.html).
+
+## Breaking changes
+
+When we make a potentially breaking change to the polyfill's API surface (like a rename) we'll continue supporting the old version, deprecated, for 3 months, and ensure that there are console warnings that a change is pending. We also announce anything that isn't a bug fix on web-animations-changes@googlegroups.com (https://groups.google.com/forum/#!forum/web-animations-changes).
```
Comment (alancutter) [inline README.md]: I wonder if this should be explicit that the 3 month deprecation is automatically handled by the polyfill and the old function will be removed even if you don't update the version of the polyfill. If you see deprecation warnings you can't avoid it by not updating.

Comment (maxh) [inline README.md]: I took a stab at making this more clear.

**Pull request merged**